oodeel/methods/base.py

```diff
@@ -95,7 +75,9 @@ def _score_tensor(self, inputs: TensorType) -> np.ndarray:
     def fit(
         self,
         model: Callable,
+        feature_layers_id: List[Union[int, str]] = [],
         fit_dataset: Optional[Union[ItemType, DatasetType]] = None,
+        input_layer_id: Optional[Union[int, str]] = None,
```

Comment:
IMO, it is more natural to have `feature_layers_id` and `input_layer_id` close together in the list of arguments. And `fit_dataset` at last, or after `model` ?